library/globals.inc.php

```diff
@@ -74,7 +74,8 @@
   $perl_bin_dir        = 'C:/xampp/perl/bin';
   $temporary_files_dir = 'C:/windows/temp';
   $backup_log_dir      = 'C:/windows/temp';
-} else {
+}
+else {
```

Comment:
You are clobbering my last commit

Reply:
also did you mean to add features back to the user menu . It has no items under it.

Reply:
Boy, did I hose this up...

I think you test this by giving me a minute to make another go...this commit was crap.

Reply:
No.  I suppose you mean in another PR.  I did not intend to re-add features.

Reply:
features has items under it...
I don't understand.

Reply:
That section is for the user preferences under miscellaneous

Reply:
Don't worry about it . I will fix it after the demos get up

Reply:
demo site is a mess.